cypress/integration/landingpage_spec.js

```diff
@@ -0,0 +1,9 @@
+describe("the landing page", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+  it("displays a list of events", () => {
+    cy.contains("Tapahtumat");
+    cy.get("div").find(".eventcard").its("length").should("be.gt", 0);
+  });
```

Comment:
Hmm, I'd use something like data-cy for this, rather than adding a redundant classname to a div. So, something like
```suggestion
    cy.get("div").get("[data-cy="event-card"]").its("length").should("be.gt", 0);
  });
```

with a proposed change to the event card.